### PR TITLE
Add In Progress section to contributing

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Add an "In Progress" section to the CONTRIBUTING.md
page of Buildah.  Once this is approved, I will open
a similar PR for Podman.

Signed-off-by: TomSweeneyRedHat <tsweeney@redhat.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -30,6 +31,16 @@ The easier it is for us to reproduce it, the faster it'll be fixed!
 
 Please don't include any private/sensitive information in your issue!
 
+## Working On Issues
+
+Once you have decided to contribute to Buildah by working on an issue, check our
+backlog of [open issues](https://github.com/containers/buildah/issues) looking
+for any that do not have an "In Progress" label attached to it.  We often times
```
Comment (rhatdan) [inline CONTRIBUTING.md:38]: How about
```
Often issues will be assigned to someone, to be worked on at a later time.  If you have the time to work on the issue now add yourself as an assignee, and set the "In Progress" label. 
```

Comment (TomSweeneyRedHat) [reply CONTRIBUTING.md]: @rhatdan I've merged this in with comments from @vrothberg  in the Google Doc and have repushed.

**Comment by rhatdan** [comment]
LGTM
@mheon @baude @vrothberg @nalind PTAL

**Review by vrothberg — COMMENTED** [review]
LGTM

**Pull request merged**